fmriprep/workflows/bold/registration.py

```diff
@@ -489,10 +496,13 @@ def init_bbreg_wf(use_bbr, bold2t1w_dof, omp_nthreads, name='bbreg_wf'):
 
         return workflow
 
-    bbregister = pe.Node(
-        BBRegisterRPT(dof=bold2t1w_dof, contrast_type='t2', registered_file=True,
-                      out_lta_file=True, generate_report=True),
-        name='bbregister', mem_gb=12)
+    if init_header:
+        bbregister = BBRegisterRPT(dof=bold2t1w_dof, contrast_type='t2', registered_file=True,
+                                   out_lta_file=True, generate_report=True, init='header')
+    else:
+        bbregister = BBRegisterRPT(dof=bold2t1w_dof, contrast_type='t2', registered_file=True,
+                                   out_lta_file=True, generate_report=True)
+    bbregister = pe.Node(bbregister, name='bbregister', mem_gb=12)
```

Comment:
This could be done instead by reverting and adding:

```Python
if init_header:
    bbregister.inputs.init = 'header'
```

That said, if we want to switch to `register` and `header`, it might end up being slightly different. Just want to draw your attention.